acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaRebalanceAT.java

```diff
@@ -325,6 +326,25 @@ public void testCommitWhenDirectAssignment() throws Exception {
         assertThat(commitStatusCode, Matchers.is(HttpStatus.NO_CONTENT.value()));
     }
 
+    @Test(timeout = 10000)
+    public void testNoConflictOnTheSamePartitionRequest() throws Exception {
+        final TestStreamingClient client1 = new TestStreamingClient(
+                URL, subscription.getId(), "batch_flush_timeout=1",
+                Optional.empty(),
+                Optional.of("{\"partitions\":[" +
+                        "{\"event_type\":\"" + eventType.getName() + "\",\"partition\":\"0\"}]}"));
+        final TestStreamingClient client2 = new TestStreamingClient(
+                URL, subscription.getId(), "batch_flush_timeout=1",
+                Optional.empty(),
+                Optional.of("{\"partitions\":[" +
+                        "{\"event_type\":\"" + eventType.getName() + "\",\"partition\":\"0\"}]}"));
+        client1.start();
+        client2.start();
+
+        waitFor(() -> assertThat(client1.getResponseCode(), Matchers.is(HttpStatus.CONFLICT.value())));
+        waitFor(() -> assertThat(client2.getResponseCode(), Matchers.is(HttpStatus.CONFLICT.value())));
```

Comment:
Do we really want _both_ clients to get a conflict response?

Reply:
that is possible when clients open connections in very close moment. session registration is not synchronized, so i can not tell which one should be dropped, so both.